packages/compass/package.json

```diff
@@ -181,6 +181,7 @@
   },
   "dependencies": {
     "@mongosh/node-runtime-worker-thread": "^2.3.0",
+    "@mongodb-js/mongodb-downloader": "^0.3.5",
```

Comment:
This will include this package with the distribution which we don't want

Reply:
Yep, that is what I was mentioning on Slack, that his should be a devDependency, compass/scripts don't have a specific package.json.

Reply:
I'm not sure what you mean, <root>/scripts is a workspace in the monorepo and does have it's own package.json at <root>/scripts/package.json

Reply:
The script that depends on the mongodb-downloader is on <root>/packages/compass/scripts/download-csfle.js, not in <root>/scripts/ and it doesn't have it's own package.json. I'm not sure why it's there, but I suppose there should be a reason.

Reply:
Ahhh, so all this time you were talking about scripts folder in compass, not scripts package, my bad. Yeah, disregard what I said about prod dependency then, this should go to the dev ones, you're correct 👍